www/styles/login.css

```diff
@@ -0,0 +1,26 @@
+.logo-budget{
```

Comment:
I recommend to use a single file for all required custom styles, also not changes in ionic.css